svgdigitizer/electrochemistry/cv.py

```diff
@@ -320,6 +433,8 @@ def rate(self):
 
         The scan rate is read from a ``<text>`` in the SVG file such as ``<text>scan rate: 50 V/s</text>``.
 
+        Besides `scan rate`, also `rate` is acceptable in the text field.
```

Comment:
It's good that this is pointed out in the documentation now.

Maybe we should only allow "scan rate" here. I don't see why that would be confusing to anyone.

Reply:
This was mostly out of convenience. I agree that it is probably more useful to only have a single option.

Reply:
fixed